README.md

```diff
@@ -212,16 +212,24 @@ bool onWatchList = amlResult.IsOnWatchList();
 
 ### .NET Core
 
-#### Requirements
-1) Install Bower with `npm install -g bower`
-1) The .NET SDK for your operating system from step no.1 ([Windows](https://www.microsoft.com/net/learn/get-started/windows) | [Linux](https://www.microsoft.com/net/learn/get-started/linux/rhel) | [MacOS](https://www.microsoft.com/net/learn/get-started/macos))
-
-#### Instructions
+#### Setup
 1) Navigate to the [src/Examples/CoreExample](src/Examples/CoreExample) folder
 1) Rename the [.env.example](src/Examples/CoreExample/.env.example) file to `.env`
 1) Fill in the environment variables in this file with the ones specific to your application (mentioned in the [Client initialisation](#client-initialisation) section)
 1) Restore the static files needed with `bower install`
 1) From the Yoti Dashboard, set the application domain to `localhost:44344` and the scenario callback URL to `/account/connect`
+
+##### With Docker
+1) Make sure the .pem file is within the `src\Examples\CoreExample` folder.
+1) Rename it to be `YotiKey.pem`
+1) `cd src`
+1) `docker-compose build --no-cache`
+1) `docker-compose up`
```

Comment:
I received the following message when running `docker-compose up`:
```
WARNING: The APPDATA variable is not set. Defaulting to a blank string.
```

Tried removing the following lines fro `docker-compose.override.yml`:
```
  - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
  - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
```

But then outputs message:
```
coreexample_1  | crit: Microsoft.AspNetCore.Server.Kestrel[0]
coreexample_1  |       Unable to start Kestrel.
```

Is there a way we can include these without using a volume on the host machine?